packages/blade/src/components/Input/SelectInput/SelectInput.tsx

```diff
@@ -103,6 +111,8 @@ const SelectInput = (props: SelectInputProps): JSX.Element => {
   );
 };
 
+const SelectInput = React.forwardRef(_SelectInput);
+// @ts-expect-error: componentId is our custom attribute
 SelectInput.componentId = 'SelectInput';
```

Comment:
oh you can actually use `WithComponentId` from utils

Reply:
Missed that one. Using that now for rest of the components. Although here its not possible to use that utility because I am also using ref and it is messing up with types when we're forwarding ref.

Tried few things but didn't quiet work so disabling the ts error for now here.